### ISSUE TITLE
Check if we can run file server in chrome/firefox extension

### ISSUE THREAD

**Issue by peterlimg** [0chain/gosdk#1252]

**Comment by cnlangzi** [comment]
@peterlimg  It should be impossible , because wasm doesn't have any tcp support. we should run it in js instead of wasm.  see
- https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb
- https://chrome.google.com/webstore/detail/live-server-web-extension/fiegdmejfepffgpnejdinekhfieaogmj

**Comment by peterlimg** [comment]
Yes, as we discussed before,  we need to check if we can run http server in extension instead.
